trestle/core/commands/create.py

```diff
@@ -39,31 +40,60 @@
 
 
 class CreateCmd(CommandPlusDocs):
-    """Create a sample OSCAL model in trestle project."""
+    """Create a sample OSCAL model in trestle project or create new elements within a given model."""
 
     name = 'create'
 
     def _init_arguments(self) -> None:
-        self.add_argument('model', help='', choices=const.MODEL_TYPE_LIST)
-        self.add_argument('-o', '--output', help='Name of the output created model.', required=True)
+        self.add_argument('-t', '--type', help='Type of model if created anew.', choices=const.MODEL_TYPE_LIST)
+        self.add_argument('-o', '--output', help='Name of the output created model.')
         self.add_argument(const.IOF_SHORT, const.IOF_LONG, help=const.IOF_HELP, action='store_true')
         self.add_argument(
             '-x', '--extension', help='Type of file output.', choices=['json', 'yaml', 'yml'], default='json'
         )
+        self.add_argument(
+            '-f', '--file', help='Optional existing OSCAL file that will have elements created within it.', type=str
+        )
+        self.add_argument(
+            '-e', '--element', help='Optional path of element to be created whithin the specified file.', type=str
+        )
 
```

Comment:
Are we using -t <model> to specify the model to be created like "create -t catalog" or the existing way "create catalog"? Any specific reason to introduce -t option?

Reply:
We need to use -t because the model type is optional.  There is no model type if we specify the file with -f.  The .json file could be anything, such as metadata.